android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -662,29 +662,33 @@ internal class BetterPlayer(
                     // LIVE
                     // Samsung devices with android 11 
                     // https://dw-ml-nfc.atlassian.net/browse/DAF-4294
-                    setQueueNavigator(object : TimelineQueueNavigator(mediaSession) {
-                        override fun getMediaDescription(
-                            player: Player,
-                            windowIndex: Int
-                        ): MediaDescriptionCompat {
-                            val extra = Bundle()
-                            extra.putString(MediaMetadataCompat.METADATA_KEY_ARTIST, author)
-                            extra.putString(MediaMetadataCompat.METADATA_KEY_TITLE, title)
-                            val mediaDescriptionCompatBuilder = MediaDescriptionCompat.Builder()
-                            mediaDescriptionCompatBuilder.apply {
-                                setExtras(extra)
-                                bitmap?.let {
-                                    setIconBitmap(it)
+                    exoPlayer?.isCurrentMediaItemLive?.let {
+                        if (it) {
```

Comment:
The code nested deeply...
And `it` is used only once.

How about checking it directly?

```
val isCurrentMediaItemLive = exoPlayer?.isCurrentMediaItemLive
if (isCurrentMediaItemLive != null && isCurrentMediaItemLive) {
    setQueueNavigator(....
```